src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -15,7 +17,11 @@ trait ConfirmableTrait
      */
     public function confirmToProceed($warning = null, $callback = null)
     {
-        $handler = $this->laravel->make(ConfirmHandler::class);
+        if ($this->laravel->bound(ConfirmHandlerContract::class)) {
```

Comment:
No. You should bind this in the console service provider.

Reply:
Ahh ok

Reply:
Can you point me at that class?

Reply:
Sorry, I meant https://github.com/laravel/framework/blob/8.x/src/Illuminate/Foundation/Console/Kernel.php.

Reply:
No worries, what's the correct function to add the bind to?